tokio/src/runtime/builder.rs

```diff
@@ -828,6 +834,36 @@ impl Builder {
         pub fn disable_lifo_slot(&mut self) -> &mut Self {
             self.disable_lifo_slot = true;
             self
+        }
+
+                /// Specifies the random number generation seed to use within all threads associated
```

Comment:
```suggestion
        /// Specifies the random number generation seed to use within all threads associated
```

formatting (rustfmt doesn't touch macro bodies :frowning: )